service-automation/src/com/appyschool/common/SoftAssertionBase.java

```diff
@@ -61,7 +61,7 @@ public void verifyTrue(Boolean b, String msg) {
 		}
 	}
 
-	public void verifyTrue(Boolean b) {
+	public static void verifyTrue(Boolean b) {
```

Comment:
why you have changed this to static?

Reply:
Without making it static, this function is not available in TestStudent.java and TestStaff.java files.